__tests__/hooks/useGroupInfo.test.js

```diff
@@ -186,6 +186,7 @@ describe('useGroupInfo should run correctly', () => {
     ]
   }));
 
+  const mockQueryMemberError = jest.fn().mockRejectedValue('error');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This mock function `mockQueryMemberError` is defined but not used in the old hunk. Make sure it is used somewhere in the tests, otherwise, it should be removed to avoid confusion.
```diff
- const mockRemoveMember = jest.fn().mockResolvedValue({
+ const mockQueryMemberError = jest.fn().mockRejectedValue('error');
  const mockRemoveMember = jest.fn().mockResolvedValue({
    members: [{user_id: '1234', name: 'elon', user: {name: 'elon'}}]
  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->